src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -605,6 +605,10 @@ public function testChgrpSymlink()
      */
     public function testChgrpSymlinkFails()
     {
+        if (defined('HHVM_VERSION')) {
+            $this->markTestSkipped('Shouldn\'t be executed in HHVM context.');
```

Comment:
This statement should be removed as we don't want to mask a test that fails for a good reason under HHVM. This will be fixed whenever we have access to the latest versions of HHVM where they fixed the problem.